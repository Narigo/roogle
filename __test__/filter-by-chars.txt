This is a sentence with a lot of characters and ultimately over 50. There is a character limit right above this sentence. So this sentence is right above that artificial limit. Small things get filtered. Longer things do not get filtered, so this is just a longer text that should appear.
