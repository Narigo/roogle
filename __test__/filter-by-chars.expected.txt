open "https://www.google.com/search?q=This%20is%20a%20sentence%20with%20a%20lot%20of%20characters%20and%20ultimately%20over"
open "https://www.google.com/search?q=So%20this%20sentence%20is%20right%20above%20that%20artificial%20limit"
open "https://www.google.com/search?q=Longer%20things%20do%20not%20get%20filtered%20so%20this%20is%20just%20a%20longer%20text%20that%20should%20appear"
